android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java

```diff
@@ -1461,6 +1461,10 @@ protected void registerForTouch(final View touchable)
 		boolean clickable = true;
 		if (proxy.hasProperty(TiC.PROPERTY_TOUCH_ENABLED)) {
 			clickable = TiConvert.toBoolean(proxy.getProperty(TiC.PROPERTY_TOUCH_ENABLED), true);
+			//If touchEnabled is specifically set, enable or disable touch sound
+			touchable.setSoundEffectsEnabled(clickable);
+		} else {
+			touchable.setSoundEffectsEnabled(false);
```

Comment:
i'd be fine to disable it for 6.0 (accept the suggested PR) and just make a ti flag in tiapp.xml which allows you to revert to previous behavior.